### ISSUE TITLE
Ore biome config should allow specific biomes to be used.

### ISSUE THREAD

**Issue by SilentChaos512** [SilentChaos512/FunOres#16]
Currently biome types are used, based on the Forge BiomeDicionary class. It would be nice if specific biomes could be listed instead.

I really can't remember why I set it up this way. It shouldn't be too difficult to change though.

**Comment by Zygus42** [comment]
By type doesn't seem a bad system either.  Maybe offer a choice in the config ?

**Comment by Zygus42** [comment]
As an addon to this suggestion, would it also be possible to have multiple listings of one resource in the config file, so for example make copper FAVORS Hills, and also make copper BLACKLIST Ocean, so that you would have more in Hills, less everywhere else, but none at all in Oceans.

**Comment by Zygus42** [comment]
I noticed that the latest version (FunOres-1.10.2-1.4.1-83) had this:
"Fixed: Biome lists for ore generation not working in some cases. The old "BiomeTypes" list will be removed from the config."

I made a new folder with nothing but funores and silentlib, and it still creates a config folder with:
     tin {
        S:BiomeListType=BLACKLIST
        S:BiomeTypes <
         >
for all metals.  I tried changing the name "S:BiomeTypes" with "S:Biomes" as per the example (which does have "S:Biomes", but it crashes:
[crash-2017-03-08_21.02.46-client.txt](https://github.com/SilentChaos512/FunOres/files/827657/crash-2017-03-08_21.02.46-client.txt)

**Comment by Zygus42** [comment]
UPDATE:  that's not why it crashed - was because I didn't have JEI installed.  May want to make a note of that on the download page.

**Comment by Zygus42** [comment]
So I tried:
    copper {
        S:BiomeListType=FAVORS
        S:Biomes <
		TAIGA
                RIVER
         >
        I:ClusterCount=10
        I:ClusterSize=8
        B:Enabled=true
        I:MaxY=255
        I:MinY=0
        I:Rarity=1
    }

and

    S:OreGenFavorsBiomeMultiplier=10

But didn't work in that there's copper spawning, but not any more in Taiga.  Is there anything wrong with this syntax ?  The example is not really very revealing.

**Comment by SilentChaos512** [comment]
I didn't know the 1.10.2 version had the JEI crash (#26). I only fixed it in 1.11. You can try Fun Ores 1.5.0 (although it probably won't work if you are using Super Multi-Drills). That's the new cross compatible build that lets 1.11 code run on 1.10.2, so it will have all recent bugfixes, but some textures and other things might be messed up.

If you decide to try the new builds, remember to update _all of my mods_, including Silent Lib or nothing will work. And backup your world before trying it, just in case.

**Comment by Zygus42** [comment]
I ran FunOres-1.11.x-1.5.0-100 & FunOres-SilentLib-1.10.2-2.0.1-41 & jei_1.10.2-3.14.7.416, as well as worldstripper for testing, no other mods.

Works fine until I change copper to:
    copper {
        S:BiomeListType=FAVORS
        S:Biomes <
minecraft:taiga
         >
        I:ClusterCount=10
        I:ClusterSize=8
        B:Enabled=true
        I:MaxY=255
        I:MinY=0
        I:Rarity=1
    }

When it crashes:
[crash-2017-03-09_18.55.43-server.txt](https://github.com/SilentChaos512/FunOres/files/830585/crash-2017-03-09_18.55.43-server.txt)

I dumped the biome names, and it definitely exists by that name and spelling.

I also tried FOREST instead of minecraft:taiga, but same crash result.
